lang/well-formed.md

```diff
@@ -217,7 +217,7 @@ impl PlaceExpr {
     fn check_wf<M: Memory>(self, locals: Map<LocalName, PlaceType>) -> Option<PlaceType> {
         use PlaceExpr::*;
         match self {
-            Local(name) => locals.get(name),
+            Local(name) => locals.get(name)?,
```

Comment:
This is again required as the other arms return `PlaceType` directly, so this arm cannot return `Option<PlaceType>`.